TangoRosStreamer/app/src/main/res/values/strings.xml

```diff
@@ -24,35 +25,20 @@
 
     <string name="save_map_dialog_title">Enter the name of the map</string>
     <string name="uuids_names_map">uuids_names_map</string>
+    <string name="ros_status">ros_status</string>
 
     <!-- Strings related to error messages -->
 
     <string name="network_error">No network found! Please check your network configuration and restart the application.</string>
-    <string name="ros_init_error">Unable to initialize ROS node. Do you have a roscore running?</string>
+    <string name="ros_init_error">Unable to connect to ROS master. Do you have a roscore running?</string>
```

Comment:
An other reason for connection failure could be that ROS master URI is incorrect. Do you agree to remove the latter part `Do you have a roscore running?` I think it could be confusing.